user/test/com/google/gwt/dev/jjs/test/Java17Test.java

```diff
@@ -121,6 +121,9 @@ public void testSwitchExprInlining() {
   public void testInlinedStringConstantsInCase() {
     assertFalse(isGwtSourceLevel17());
   }
+  public void testCaseArrowLabelsVoidExpression() {
```

Comment:
Missing newline above.

Reply:
Also missing in main (this is a backport for 2.12.2), so I'm somewhat unconcerned.

Related-ish, I will put up a PR soon that might get our checkstyle annotations back, again, in PRs...